src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -91,6 +91,8 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | CSKY         | `freg`                          | None           | `f32`,                                  |
 | SPARC        | `reg`                           | None           | `i8`, `i16`, `i32`, `i64` (SPARC64 only) |
 | SPARC        | `yreg`                          | N/A            | Only clobbers                           |
+| LoongArch32  | `reg`                           | None           | `i8`, `i16`, `i32`, `f32`               |
+| LoongArch32  | `freg`                          | None           | `f32`, `f64`                            |
```

Comment:
The code above reference `LoongArch64` but this table doesn't. Is that a mistake?

Reply:
Could you also add LoongArch32 to the table above?

> The code above reference `LoongArch64` but this table doesn't. Is that a mistake?

LoongArch64 assembly is already stable. This PR adds it for LoongArch32.

Reply:
Done.